apps/web/src/pages/PrivacyPolicy/PrivacyPolicy.tsx

```diff
@@ -0,0 +1,220 @@
+          : "https://laws-lois.justice.gc.ca/fra/lois/P-33.01/section-15.html"
+      }
+    >
+      {chunks}
+    </Link>
+  );
+  const justiceLaws29Link = (chunks: React.ReactNode) => (
+    <Link
+      newTab
+      external
+      href={
+        locale === "en"
+          ? "https://laws-lois.justice.gc.ca/eng/acts/P-33.01/section-29.html"
+          : "https://laws-lois.justice.gc.ca/fra/lois/P-33.01/section-29.html"
+      }
+    >
+      {chunks}
+    </Link>
+  );
+
+  const justiceLaws30Link = (chunks: React.ReactNode) => (
+    <Link
+      newTab
+      external
+      href={
+        locale === "en"
+          ? "https://laws-lois.justice.gc.ca/eng/acts/P-33.01/section-30.html"
+          : "https://laws-lois.justice.gc.ca/fra/lois/p-33.01/section-30.html"
+      }
+    >
+      {chunks}
+    </Link>
+  );
+
+  const publicServiceLink = (chunks: React.ReactNode) => (
+    <Link
+      newTab
+      external
+      href={
+        locale === "en"
+          ? "https://www.canada.ca/en/public-service-commission/services/oversight-activities/investigations.html"
+          : "https://www.canada.ca/fr/commission-fonction-publique/services/activites-surveillance/enquetes.html"
+      }
+    >
+      {chunks}
+    </Link>
+  );
+
+  const personalInfoLink = (chunks: React.ReactNode) => (
+    <Link
+      newTab
+      external
+      href={
+        locale === "en"
+          ? "https://www.canada.ca/en/treasury-board-secretariat/corporate/transparency/treasury-board-secretariat-sources-federal-government-employee-information-info-source.html"
+          : "https://www.canada.ca/fr/secretariat-conseil-tresor/organisation/transparence/secretariat-conseil-tresor-sources-renseignements-gouvernement-federal-fonctionnaires-federaux-info-source.html"
+      }
+    >
+      {chunks}
+    </Link>
+  );
+
+  const id = "privacy";
+  const pageTitle = intl.formatMessage({
+    defaultMessage: "Privacy policy",
+    id: "cYNDhP",
+    description: "Title for the websites privacy policy",
+  });
+
+  const subtitle = intl.formatMessage({
+    defaultMessage:
+      "Learn more about how GC Digital Talent handles user privacy.",
+    id: "Vz719J",
+    description: "Sub title for the websites privacy policy",
+  });
+
+  const crumbs = useBreadcrumbs([
+    {
+      label: pageTitle,
+      url: paths.privacyPolicy(),
+    },
+  ]);
+
+  return (
+    <>
+      <Hero title={pageTitle} subtitle={subtitle} crumbs={crumbs} />
+      <div data-h2-container="base(center, large, x1) p-tablet(center, large, x2)">
+        <TableOfContents.Wrapper data-h2-margin-top="base(x3)">
+          <TableOfContents.Navigation>
+            <TableOfContents.List>
+              <TableOfContents.ListItem key={id}>
+                <TableOfContents.AnchorLink id={id}>
+                  {pageTitle}
+                </TableOfContents.AnchorLink>
+              </TableOfContents.ListItem>
+            </TableOfContents.List>
```

Comment:
So, this kind of seems redundant. Do we need a TOC for a single section? 🤔 I know it is in the design but the design also has no heading and that is usually what we link to 😖

Reply:
@yonikid15 @esizer 

Yeah I was confused as to how to handle this as well. We can get rid of it for now, though I don't want the content to stretch the full width either. Maybe leverage the `h2-container` attribute to keep it within legible line lengths,

Reply:
Okay sounds good! Fixed by [5d86c61](https://github.com/GCTC-NTGC/gc-digital-talent/pull/9143/commits/5d86c612ee5b3acc9b1f541179cf29e77da5585b)